src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Menu/Builder.php

```diff
@@ -37,8 +44,14 @@ public function mainMenu(array $options)
             return $menu;
         }
 
-        $menu->addChild('Services', ['route' => 'service_overview']);
-
+        if ($this->authorizationService->isAdministrator() && $this->authorizationService->getActiveServiceId()) {
+            $menu->addChild('Service overview', [
+                'route' => 'service_admin_overview',
+                'routeParameters' => ['serviceId' => $this->authorizationService->getActiveServiceId()],
```

Comment:
What happens with the activeServiceId if your session is timing out?

Reply:
It is lost just like I would expect it to be. Do you see a problem I do not?